src/components/modules/AverageTimeReports/AverageTimeReports.tsx

```diff
@@ -0,0 +1,92 @@
+/*eslint-disable*/
```

Comment:
```suggestion
```
Don't disable eslint at all. The purpose of eslint is to help write good code, so disabling it is never a way to go.